tests/python_client/bulk_insert/test_bulk_insert.py

```diff
@@ -73,7 +73,7 @@ def teardown_method(self, method):
 class TestBulkInsert(TestcaseBaseBulkInsert):
 
     @pytest.mark.tags(CaseLabel.L3)
-    @pytest.mark.parametrize("is_row_based", [True, False])
+    @pytest.mark.parametrize("is_row_based", [True])
```

Comment:
why not remove this parameter directly?

Reply:
it used for generating data files, and i assume that we would support column_based in the coming future

Reply:
OK